addon/components/nrg-checkbox.js

```diff
@@ -1,31 +1,18 @@
-import { computed } from '@ember/object';
-import { alias, bool } from '@ember/object/computed';
-import Component from '@ember/component';
-import layout from '../templates/components/nrg-checkbox';
-import Validation from 'ember-nrg-ui/mixins/validation';
+import { action } from '@ember/object';
+import { guidFor } from '@ember/object/internals';
+import { tracked } from '@glimmer/tracking';
+import NrgValidationComponent from './nrg-validation-component';
 
-export default Component.extend(Validation, {
-  layout,
-  classNames: ['ui', 'checkbox'],
-  classNameBindings: ['type', 'disabled', 'fitted', 'readonly:read-only'],
+export default class NrgCheckboxComponent extends NrgValidationComponent {
+  @tracked
+  focusId = 'field-' + guidFor(this);
 
-  checked: alias('value'),
-  _checked: bool('checked'),
-  bindValue: true,
+  get checked() {
+    return !!this.value;
+  }
```

Comment:
Can you use the `@bool` computed property here?
```suggestion
  @bool('value')
  checked;
```

Reply:
Using a computed property results in this linting error:
![image](https://user-images.githubusercontent.com/2756533/139873673-244b17ae-6db3-4bd9-89a5-997df33e4295.png)

Reply:
That's very strange... the [official API docs](https://api.emberjs.com/ember/3.28/functions/@ember%2Fobject%2Fcomputed/bool) show the decorator version for a native class, but the [eslint rule](https://github.com/ember-cli/eslint-plugin-ember/blob/master/docs/rules/no-computed-properties-in-native-classes.md) says to not use them. In that case, I'm good with your version.